src/sentry/rules/processing/processor.py

```diff
@@ -149,18 +149,23 @@ def activate_downstream_actions(
     event: GroupEvent,
     notification_uuid: str | None = None,
     rule_fire_history: RuleFireHistory | None = None,
+    is_post_process: bool | None = None,
```

Comment:
cc @iamrajjoshi Actions in workflow engine will also use `activate_downstream_actions`, and i only want to fire metrics if we're using this function in the existing system. is this assumption correct?

Reply:
update; no we are not